html/pfappserver/lib/pfappserver/Base/Form.pm

```diff
@@ -171,6 +176,47 @@ sub id_validator {
    };   
 }
 
+before 'process' => sub {
+    my ($self) = @_;
+    foreach my $field (@{$self->fields}){
+        # Populate all the FingerbankSelect fields with the proper values extracted from the database.
+        if($field->type eq "FingerbankSelect"){
```

Comment:
Ok I understand now.
Try and override _inner_validate_field or validate_field in FingerbankSelect.
Look at /usr/share/perl5/vendor_perl/HTML/FormHandler/Validate.pm and /usr/share/perl5/vendor_perl/HTML/FormHandler/Field/Select.pm
To see how it is implemented.